src/test/java/hudson/plugins/git/GitStatusTest.java

```diff
@@ -178,6 +184,47 @@ public void testDoNotifyCommitWithTwoBranchesAndAdditionalParameter() throws Exc
 
     }
 
+    @DataPoints("branchSpecPrefixes")
+    public static String[] branchSpecPrefixes = new String[] {
+            "",
+            "refs/remotes/",
```

Comment:
No wildcard cases and no regex cases?  Is that because the wild cards and the regular expressions are already expected to be expanded?